### PR TITLE
Added force application activation on macosx with qt 5.5.1

### PR DESCRIPTION
**Pull request by mileschet**
I was getting the following errors when trying to build using MacOSX Catalina 10.15.4:

CXX      qt/libbitcoinqt_a-guiutil.o
qt/guiutil.cpp:364:14: error: no matching function for call to 'objc_msgSend'
    id app = objc_msgSend((id) objc_getClass("NSApplication"), sel_registerName("sharedApplication"));
             ^~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/objc/message.h:63:1: note:
      candidate function not viable: requires 0 arguments, but 2 were provided
objc_msgSend(void /* id self, SEL op, ... */ )
^
qt/guiutil.cpp:365:5: error: no matching function for call to 'objc_msgSend'
    objc_msgSend(app, sel_registerName("activateIgnoringOtherApps:"), YES);
    ^~~~~~~~~~~~
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/objc/message.h:63:1: note:
      candidate function not viable: requires 0 arguments, but 3 were provided
objc_msgSend(void /* id self, SEL op, ... */ )
^
2 errors generated.

Basically i was able to build getting the changes from the latest bitcoin core source code.

Please let me know if there is anything else i need/should do ?

### CODE REVIEW

**Comment by stevenroose** [comment]
Hi, thanks for the contribution! Could you perhaps find a/the commit where this change was made and/or your error was resolved in upstream Bitcoin Core?

**Comment by mileschet** [comment]
Hi, thank you here is the commit from bitcoin core https://github.com/bitcoin/bitcoin/commit/0bb33b5348dbddd65b88a7f00449c965562355d3
